src/create_update.c

```diff
@@ -227,6 +227,25 @@ static int check_build_env(void)
 	return 0;
 }
 
+static int check_group_file(void)
+{
+	int ret = -1;
+
+	// Ensure the os-core group is defined
+	while (1) {
```

Comment:
Sure, will do. while (1) is pretty common throughout the code base and I was trying to match implicit style but I'll fix this and rebase.

Thanks for the feedback!